docs/source/conf.py

```diff
@@ -91,7 +91,17 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = []
+exclude_patterns = [
+        'datatypes/**',
+        'developer_guide/**',
+        'get_started/**',
+        'howto/installation_more/index.rst',
```

Comment:
I think Giovanni added this one, do you remember what the idea here was? Where should this be added at some point?

Reply:
We didn't know where to put it.